runtime/types

```diff
@@ -1,27 +0,0 @@
-#!/bin/sh
-set -euo pipefail
-
-#
-# Update the version and run to update ./types.d.ts
-#
-
-# See latest version: https://www.npmjs.com/package/@types/aws-lambda
-# Note: DefinitelyTyped is MIT License.
-curl -fSsL https://unpkg.com/@types/aws-lambda@8.10.39/index.d.ts -o types.d.ts
```

Comment:
So, updating here no longer worked?

Reply:
No, it is not a single file anymore, rather a folder full of files that reference each other.